CMakeLists.txt

```diff
@@ -239,7 +239,7 @@ if (NOT USE_PRE_BUILT_NGRAPH)
     ExternalProject_Add(
         ext_ngraph
         GIT_REPOSITORY https://github.com/NervanaSystems/ngraph
-        GIT_TAG v0.18.0
+        GIT_TAG v0.19.0-rc.1
```

Comment:
We cannot move to the new tag as there are crashes that are being debugged now. Please remove these as there's a separate PR that's already doing this.

Reply:
This PR should not be merged until the bugs in v0.19.0 is fixed. And it needs v0.19.0 to build